app/components/ui/set-up-domain/progress-header/index.js

```diff
@@ -0,0 +1,37 @@
+// External dependencies
+import i18n from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+// Internal dependencies
+import { getPath } from 'routes';
+import ProgressBar from 'components/ui/progress-bar';
+import styles from './styles.scss';
+
+const ProgressHeader = ( { children, progress } ) => {
+	return (
+		<div className={ styles.headerContainer }>
+			<div className={ styles.header }>
+				<h1 className={ styles.headerText }>
+					<span className={ styles.setUpLabel }>
+						{ i18n.translate( 'Setup: ' ) }
+					</span>
+					{ children }
+				</h1>
+				<div className={ styles.progressHeader }>
+					<div className={ styles.fullWidth }>
+						<ProgressBar progress={ progress } />
+					</div>
+					<a className={ styles.exit } href={ getPath( 'myDomains' ) }>Exit</a>
+				</div>
+			</div>
+		</div>
+	);
+};
+
+ProgressHeader.propTypes = {
+	children: PropTypes.string,
```

Comment:
Isn't this prop mandatory? We should probably make it required.

We may also want to declare this prop as [`node`](https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes) since it can be used to pass formatted text (e.g. a text with some parts embedded in a `<strong />`):

```
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.
optionalNode: React.PropTypes.node,
```